test/README.md

```diff
@@ -290,8 +290,11 @@ Array of IPV6 hosts.
 ### mustCall(fn[, expected])
 * fn [&lt;Function>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function)
 * expected [&lt;Number>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Number_type) default = 1
+* return [&lt;Function>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function)
 
-Number of times `fn` should be called.
+Returns a function that calls `fn`. If the returned function has not been called
+exactly `expected` number of times when the test is complete, then the test will
```

Comment:
We may want to specify that:

1. It checks during `process.on('exit')`
2. Wrapping `exit` handlers causes problems.